packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/theme.js

```diff
@@ -17,18 +17,32 @@ const theme = {
       }
     },
     extend: props => {
-      const { theme: { dark, global: { colors } } } = props
+      const { theme: { global: { colors } }, completeness } = props
+      const percentComplete = `${parseInt(completeness * 100)}%`
```

Comment:
do you need to do this again? if so, it needs the radix parameter again

Reply:
Good point. Fixed now.